src/components/Dashboard/StaffNotation.vue

```diff
@@ -154,16 +174,38 @@
 
 <script lang="ts">
 import Vue from 'vue'
-import { Component, Prop, Emit } from 'vue-property-decorator'
-import { Getter } from 'vuex-class'
+import { Component, Emit, Prop } from 'vue-property-decorator'
 import { navigate } from '@/utils'
-import { DissolutionTypes, DissolutionNames, FilingTypes, FilingNames } from '@/enums'
+import { ApplicationTypes, DissolutionNames, DissolutionTypes, FilingNames, FilingTypes, RestorationTypes } from '@/enums'
 import { AddStaffNotationDialog } from '@/components/dialogs'
 import { FilingMixin } from '@/mixins'
 import { LegalServices } from '@/services'
+import { mapGetters } from 'vuex'
 
 @Component({
+  computed: {
+    ...mapGetters([
+      'getIdentifier',
+      'isAdminFreeze',
+      'isBenBcCccUlc',
+      'isCoop',
+      'isEntityInLimitedRestoration',
+      'isFirm',
+      'isHistorical'
+    ])
+  },
   components: { AddStaffNotationDialog },
+  computed: {
+    ...mapGetters([
+      'getIdentifier',
+      'isAdminFreeze',
+      'isBenBcCccUlc',
+      'isCoop',
+      'isEntityInLimitedRestoration',
+      'isFirm',
+      'isHistorical'
+    ])
+  },
```

Comment:
Duplicate code.

Reply:
Removed

Reply:
Not sure why Github isn't marking this as outdated, but it's been fixed.